x-pack/plugin/transform/qa/single-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/transform/integration/TransformRestTestCase.java

```diff
@@ -477,6 +495,13 @@ protected static String getTransformState(String transformId) throws IOException
         return (Map<?, ?>) transforms.get(0);
     }
 
+    protected static Map<String, Object> getTransformStateAndStats(int from, int size) throws IOException {
```

Comment:
Since it returns a number of transforms, would it make sense to name it `getTransformsStateAndStats`?